pierone/api.py

```diff
@@ -180,82 +247,100 @@ def load_docker_config():
-        access_token = get_token(name, ['uid', 'application.write'])
+        access_token = get_token(name, ["uid", "application.write"])
     docker_login_with_token(url, access_token)
 
 
 def docker_login_with_token(url, access_token):
-    '''Configure docker with existing OAuth2 access token'''
+    """Configure docker with existing OAuth2 access token"""
     warnings.warn("deprecated", DeprecationWarning)
     dockercfg = load_docker_config()
-    basic_auth = codecs.encode('oauth2:{}'.format(access_token).encode('utf-8'), 'base64').strip().decode('utf-8')
-
-    dockercfg['auths'] = dockercfg.get('auths', {})
-    dockercfg['auths'][url] = {'auth': basic_auth,
-                               'email': 'no-mail-required@example.org'}
+    basic_auth = (
+        codecs.encode("oauth2:{}".format(access_token).encode("utf-8"), "base64")
+        .strip()
+        .decode("utf-8")
+    )
+
+    dockercfg["auths"] = dockercfg.get("auths", {})
+    dockercfg["auths"][url] = {
+        "auth": basic_auth,
+        "email": "no-mail-required@example.org",
+    }
 
     # Explicitly disable credential helpers for the host in URL
-    dockercfg['credHelpers'] = dockercfg.get('credHelpers', {})
+    dockercfg["credHelpers"] = dockercfg.get("credHelpers", {})
     hostname = urlparse(url).hostname
-    dockercfg['credHelpers'][hostname] = ""
+    dockercfg["credHelpers"][hostname] = ""
 
     store_docker_config(dockercfg)
 
 
 def iid_auth():
-    '''Return AWS instance identity document encoded as a Pier One atuh token'''
-    pkcs7 = request('http://169.254.169.254', '/latest/dynamic/instance-identity/pkcs7')
-    basic_auth = 'instance-identity-document:{}'.format(pkcs7.text).encode('utf-8')
-    return base64.b64encode(basic_auth).decode('utf-8')
+    """Return AWS instance identity document encoded as a Pier One atuh token"""
+    pkcs7 = request("http://169.254.169.254", "/latest/dynamic/instance-identity/pkcs7")
+    basic_auth = "instance-identity-document:{}".format(pkcs7.text).encode("utf-8")
+    return base64.b64encode(basic_auth).decode("utf-8")
 
 
 def docker_login_with_iid(url):
-    '''Configure docker with IID auth'''
+    """Configure docker with IID auth"""
     dockercfg = load_docker_config()
 
-    if 'auths' not in dockercfg:
-        dockercfg['auths'] = {}
-    dockercfg['auths'][url] = {'auth': iid_auth(),
-                               'email': 'no-mail-required@example.org'}
+    if "auths" not in dockercfg:
+        dockercfg["auths"] = {}
+    dockercfg["auths"][url] = {
+        "auth": iid_auth(),
+        "email": "no-mail-required@example.org",
+    }
 
     store_docker_config(dockercfg)
 
 
-def request(url, path, access_token: str = None,
-            not_found_is_none: bool = False, method: str = 'GET', data=None) -> requests.Response:
+def request(
+    url,
+    path,
+    access_token: str = None,
+    not_found_is_none: bool = False,
+    method: str = "GET",
+    data=None,
+) -> requests.Response:
```

Comment:
```suggestion
) -> Optional[requests.Response]:
```
We have [return None](https://github.com/zalando-stups/pierone-cli/pull/97/files#diff-54b230de210663e741305792c3396e44ab9165a0fd1819e9bf5057bb1a2543dfR346)